javalib/src/main/scala/java/nio/channels/spi/AbstractSelectableChannel.scala

```diff
@@ -0,0 +1,94 @@
+package java.nio.channels.spi
+
+import java.nio.channels.{
```

Comment:
I believe the recommended style is to use "import java.nio.channels._"
when more than about 3 symbols are being imported.  This comes 
from the various `scala-lang.org` style guide.

I think the '_' in the import will continue to work with Scala 3 for
awhile longer (I think it is '?' there). If it is '?' in an import for Scala 3 and
they stop supporting it, Scala Native, and other projects which support both Scala 2 and 3,  will have to change things all over the place.


Most times in a testing environment I want a "known closed world"
and import only the classes I use in the test. This means I go over
the three or so recommendation.